.claude/skills/tech-news-digest/SKILL.md

```
@@ -314,12 +313,12 @@ Slack mrkdwn 記法 (`*bold*`, `_italic_`, `<URL|label>`)。30000 文字以内 (
 **記事が 1 件以上ある場合**:
 
 - 冒頭にレポート概要 1〜2 文
-- 「カテゴリ別ハイライト (bigtech / ai / jp)」「注目記事 (各記事は 1 行、リンク付き)」を Slack で読みやすく整形
+- 「カテゴリ別ハイライト (bigtech / ai)」「注目記事 (各記事は 1 行、リンク付き)」を Slack で読みやすく整形
 
 **記事が 0 件の場合**:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)